test/e2e/traceflow_test.go

```diff
@@ -151,46 +152,48 @@ func testTraceflowIntraNodeANNP(t *testing.T, data *TestData) {
 		{
 			name:      "ANNPDenyIngressIPv4",
 			ipVersion: 4,
-			tf: &v1alpha1.Traceflow{
+			tf: &v1beta1.Traceflow{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: randName(fmt.Sprintf("%s-%s-to-%s-%s-", data.testNamespace, node1Pods[0], data.testNamespace, node1Pods[1])),
 				},
-				Spec: v1alpha1.TraceflowSpec{
-					Source: v1alpha1.Source{
+				Spec: v1beta1.TraceflowSpec{
+					Source: v1beta1.Source{
 						Namespace: data.testNamespace,
 						Pod:       node1Pods[0],
 					},
-					Destination: v1alpha1.Destination{
+					Destination: v1beta1.Destination{
 						Namespace: data.testNamespace,
 						Pod:       node1Pods[1],
 					},
-					Packet: v1alpha1.Packet{
-						IPHeader: v1alpha1.IPHeader{
+					Packet: v1beta1.Packet{
+						IPHeader: &v1beta1.IPHeader{
 							Protocol: protocolTCP,
 						},
-						TransportHeader: v1alpha1.TransportHeader{
-							TCP: &v1alpha1.TCPHeader{
+						TransportHeader: v1beta1.TransportHeader{
+							TCP: &v1beta1.TCPHeader{
 								DstPort: 80,
 								SrcPort: 10000,
-								Flags:   2,
+								Flags:   &tcpFlags,
 							},
 						},
 					},
 				},
 			},
-			expectedPhase: v1alpha1.Succeeded,
-			expectedResults: []v1alpha1.NodeResult{
+			expectedPhase: v1beta1.Succeeded,
+			expectedResults: []v1beta1.NodeResult{
 				{
 					Node: node1,
-					Observations: []v1alpha1.Observation{
+					Observations: []v1beta1.Observation{
 						{
-							Component: v1alpha1.ComponentSpoofGuard,
-							Action:    v1alpha1.ActionForwarded,
+							Component: v1beta1.ComponentSpoofGuard,
+							Action:    v1beta1.ActionForwarded,
 						},
 						{
-							Component:     v1alpha1.ComponentNetworkPolicy,
-							ComponentInfo: "IngressMetric",
-							Action:        v1alpha1.ActionDropped,
+							Component:         v1beta1.ComponentNetworkPolicy,
+							ComponentInfo:     "IngressMetric",
+							Action:            v1beta1.ActionDropped,
+							NetworkPolicy:     "AntreaNetworkPolicy:testtraceflow-.{8}/test-annp-deny-ingress",
```

Comment:
Do we really need a regex here? We know what the name of the Namespace is by the time we declare the test cases.